ovn-tester/ovn_workload.py

```diff
@@ -117,10 +122,15 @@ def start(self, cluster_cfg):
     @ovn_stats.timeit
     def connect(self, cluster_cfg):
         print(f'***** connecting worker {self.container} *****')
+        if not cluster_cfg.enable_ssl:
+            node_remote = \
+                'tcp:192.16.0.1:6642,tcp:192.16.0.2:6642,tcp:192.16.0.3:6642'
```

Comment:
I think the best way to do this is to actually change calculate_default_node_remotes in ovn_tester.py and pass it enable_ssl based on which it should either set tcp or ssl as protocol [here](https://github.com/dceara/ovn-heater/blob/6aeafc0ec64296cf7f018700fed9df8904fa482d/ovn-tester/ovn_tester.py#L59).